src/protocols/secure_channel/SessionEstablishmentExchangeDispatch.cpp

```diff
@@ -34,15 +32,9 @@ CHIP_ERROR SessionEstablishmentExchangeDispatch::SendMessageImpl(SecureSessionHa
                                                                  System::PacketBufferHandle && message,
                                                                  EncryptedPacketBufferHandle * retainedMessage)
 {
-    ChipLogProgress(Ble, "SessionEstablishmentExchangeDispatch::SendMessageImpl mBLETransport %p, mTransportMgr %p", mBLETransport,
-                    mTransportMgr);
+    ChipLogProgress(Ble, "SessionEstablishmentExchangeDispatch::SendMessageImpl  mTransportMgr %p", mTransportMgr);
```

Comment:
Should we update the log tag to something more generic? Maybe Inet, or ExchangeManager.